forge-gui/res/cardsfolder/upcoming/agent_of_the_shadow_thieves.txt

```diff
@@ -0,0 +1,10 @@
+Name:Agent of the Shadow Thieves
+ManaCost:1 B
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddTrigger$ AttacksPlayer | Description$ Commander creatures you own have "Whenever this creature attacks a player, if no opponent has more life than that player, put a +1/+1 counter on this creature. It gains deathtouch and indestructible until end of turn."
+SVar:AttacksPlayer:Mode$ Attacks | ValidCard$ Card.Self | Attacked$ Player | Execute$ TrigPutCounter | Condition$ AttackedOpponentWithMostLife | TriggerDescription$ Whenever this creature attacks a player, if no opponent has more life than that player, put a +1/+1 counter on this creature. It gains deathtouch and indestructible until end of turn.
```

Comment:
is the "Attacked$" Param checked for intervening If Clause?

if yes then shouldn't Opponent.withMostLife work too?

It needs to be checked how this works when the Commander is controlled by another player how it is handled there?

Reply:
Looks like it probably would work:
![image](https://user-images.githubusercontent.com/103371817/171049441-1505b5e9-bcde-4853-8964-dafac57acd29.png)

Reply:
Dethrone is not intervening if, so no proof

Reply:
I see

I tested Agent of the Shadow Thieves with `Attacked$ Opponent.withMostLife` and it didn't trigger when attacking an opponent with less life

What else should I test there?

Reply:
check if it does do another check again

attack opponent with most life so it does trigger, then while trigger is on the stack, reduce life so it isn't most life anymore


How does it work with team/allies? if you or your team mate has the most life, but you attack the opponent which has most life (from opponent players) would it trigger? I think it should, right? should probably withMostLife be extended to "withMostLife Opponent" ?

Reply:
I'll check these things

Reply:
Ugh

So if I have two opponents with the same amount of life and attack one it doesn't trigger. :(